test/rulesets/Base/mapreduce.jl

```diff
@@ -238,6 +238,21 @@ const CFG = ChainRulesTestUtils.ADviaRuleConfig()
 end
 
 @testset "Accumulations" begin
+    @testset "cumsum" begin
+        v = round.(10 .* randn(9), sigdigits=3)
+        m = round.(10 .* randn(4, 5), sigdigits=3)
```

Comment:
```suggestion
        v = round.(10 .* randn(9), digits=3)
        m = round.(10 .* randn(4, 5), digits=3)
```

`digits` produces nicer output than `sigdigits` as sigdigits can have exponents.
and summing `0.123` with `0.123e-7` makes an ugly number

Reply:
I don't even remember why I cared, I guess I copied this long ago from some other PR where there was more involved println debugging?